src/app/app-routing.module.ts

```diff
@@ -4,9 +4,9 @@
 import {NgModule} from '@angular/core';
 import {RouterModule, Routes} from '@angular/router'
 
-import {CohortBuilderComponent} from './cohort-builder.component'
-import {LoginComponent} from './login.component'
-import {SelectRepositoryComponent} from './select-repository.component'
+import {CohortBuilderComponent} from 'app/views/cohort-builder/cohort-builder.component'
```

Comment:
This naming has some redundancy. Can you just call it `../cohort-builder/component`?